Cargo.lock

```diff
@@ -1045,7 +1045,7 @@ dependencies = [
 [[package]]
 name = "matrix-sdk-common"
 version = "0.8.0"
-source = "git+https://github.com/matrix-org/matrix-rust-sdk?branch=release-for-crypto-wasm-12#37c17cf854a70fe6f719c7fde49a6b0e4402988f"
+source = "git+https://github.com/matrix-org/matrix-rust-sdk#e99939db857ca36054c0023ef68cc181236fdee4"
```

Comment:
can we do this without rolling back the rust-sdk? ie, just:
```suggestion
source = "git+https://github.com/matrix-org/matrix-rust-sdk#37c17cf854a70fe6f719c7fde49a6b0e4402988f"
```

(likewise below)

Reply:
Good idea. I force-pushed since adding a commit would be more confusing imo.